messages/maintainer_access.md

```diff
@@ -0,0 +1,19 @@
+Hello {NAME},
+
+You have been nominated as {role} and the Coordinating Committee enthusiastically accepted this nomination! We would be thrilled if you are able to accept this nomination. {More context might be necessary for some people - fill it in here}
+
+You can see the formal description of the maintainers role at http://www.astropy.org/team.html. If you have any uncertainty or if there is extended debate about merging a PR, seek consensus from other maintainers in the Project, or if absolutely necessary, the Coordination Committee.
+
+If you would like to accept this nomination, we ask that you familiarize yourself with the following Project guidelines:
+
+* [Developer guidelines for Astropy](https://docs.astropy.org/en/latest/#developer-documentation)
+* [Basic development workflow](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html)
+* [Guidelines on when to squash or rebase](https://docs.astropy.org/en/latest/development/when_to_rebase.html)
+* [Astropy Project Code of Conduct](https://www.astropy.org/code_of_conduct.html)
+
```

Comment:
We discussed on the CoCo call that it might also make sense to add a statement to encourage nominees / new maintainers to join the monthly dev telecons. What do you think @eteq?

Reply:
Agreed, added in commit I'm about to push up.